apps/bidsificator/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:24.04
+SHELL ["/bin/bash", "-xe", "-o", "pipefail", "-c"]
+
+ARG APP_NAME
+ARG APP_VERSION
+
+WORKDIR /apps/${APP_NAME}
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy \
+    software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy \
+    git python3.10 python3.10-venv python3.10-dev python3-pip \
+    # lib necessary to have a working pyqt application \
+    libxcb-icccm4 libxcb-randr0 libxcb-render-util0 \
+    libxcb-shape0 libxcursor1 libxkbcommon-x11-0 \
+    libxcb-keysyms1 libglib2.0-0 libdbus-1-3 libxcb-icccm4 \
+    libxcb-randr0 libxcb-render-util0 libxcb-shape0 fontconfig \
+    libfontconfig1 libxcb-image0 libxcb-util1 && \
+    git clone https://github.com/HIP-infrastructure/Bidsificator.git && \
```

Comment:
`ADD` (or `COPY`) supports git projects as well, or their archive via HTTP.

Reply:
Good to know 🫡